pusher_push_notifications/__init__.py

```diff
@@ -117,6 +127,14 @@ def _make_request(self, method, path, path_params, body=None):
         url = _make_url(scheme='https', host=self.endpoint, path=path)
 
         session = requests.Session()
+        # We've had multiple support requests about this library not working
+        # on PythonAnywhere (a popular python deployment platform)
+        # They require that proxy servers be loaded from the environment when
+        # making requests (on their free plan).
+        # This flag enables this behaviour.
+        if self._use_proxy_env_vars:
```

Comment:
So instead of making a new flag, can't we just have more specific env vars? If we use something like `BEAMS_CLIENT_HTTP_PROXY`, we don't really need to create this flag because the env var is already specific enough to only apply to this SDK.

If this is a bit awkward as these ones are already set automatically, perhaps we can consider just using them by default? Does `urllib3`, `requests` and `curl` packages require this second flag?